src/utils/text.test.ts

```diff
@@ -0,0 +1,35 @@
+import { describe, expect, it } from 'vitest';
+import { transformToSentenceCase } from './text';
+
+describe('Text transformation utilities', () => {
+  describe('skipTrimToSentenceCase', () => {
+    const normalCases = [
+      { input: 'ION ORCHARD MALL', expected: 'Ion orchard mall' },
+      { input: 'FAR EAST PLAZA', expected: 'Far east plaza' },
+      { input: 'MANDARIN GALLERY', expected: 'Mandarin gallery' },
+    ];
+
+    const edgeCases = [
```

Comment:
Rather than grouping edgeCases in one, best practice is to name each case specifically as individual tests!